cmd/cluster/create/runner.go

```diff
@@ -0,0 +1,57 @@
+package create
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cluster/config"
+	"sigs.k8s.io/kind/pkg/cluster/create"
+)
+
+const (
+	waitForReady = 2 * time.Minute
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) RunWithError(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
+
+	err := r.flag.Validate()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = r.run(ctx, cmd, args)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
+	var err error
+
+	{
+		kindCtx := cluster.NewContext(r.flag.Name)
+		cfg := &config.Cluster{}
```

Comment:
This is simple for now. Next PR will define nodes and use our own retagged image instead of https://hub.docker.com/r/kindest/node/tags